tests/lib/prepare-restore.sh

```diff
@@ -394,7 +394,9 @@ prepare_project_each() {
 prepare_suite() {
     # shellcheck source=tests/lib/prepare.sh
     . "$TESTSLIB"/prepare.sh
-    if [[ "$SPREAD_SYSTEM" == ubuntu-core-16-* ]]; then
+    if [[ "$SPREAD_SYSTEM" == ubuntu-core-18-* ]]; then
+        prepare_core18
```

Comment:
Can we keep some consistency between prepare_core18 and prepare_all_snap (maybe rename _all_snap to prepare_core_legacy or similar